misc/python/materialize/version_consistency/ignore_filter/version_consistency_ignore_filter.py

```diff
@@ -96,6 +97,18 @@ def shall_ignore_expression(
         ):
             return YesIgnore("Accepted: no order explicitly specified")
 
+        if (
+            self.lower_version < MZ_VERSION_0_81_0 <= self.higher_version
+            and expression.matches(
+                partial(
+                    matches_fun_by_any_name,
+                    function_names_in_lower_case={"min", "max"},
+                ),
+                True,
+            )
+        ):
+            return YesIgnore("Implemented min/max for interval and time types in 24007")
```

Comment:
So this is just a temporary fix, for this specific version, right?

Reply:
The way that the filter works is that it only ignores this inconsistency when you test < 0.81 against >= 0.81. So yes.